### PR TITLE
Update launch method `aprun`

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: src/radical/pilot/agent/launch_method/aprun.py [hunk 1]
```diff
@@ -1,221 +1,84 @@
...
     # --------------------------------------------------------------------------
     #
-    def __init__(self, name, cfg, session):
-
-        LaunchMethod.__init__(self, name, cfg, session)
+    def _configure(self):
 
+        self.launch_command = ru.which('aprun')
+        # only one instance of `aprun` is allowed per node at any time,
+        # otherwise multiple instances will be executed sequentially
```
Comment (andre-merzky) [inline src/radical/pilot/agent/launch_method/aprun.py:22]: Do you know if this universally true for aprun, or is that  a system config setting on Theta?

**Review by andre-merzky — COMMENTED** [review]
The PR seems to change the LM to ignore the scheduler decision (`slots` is not evaluated).  Is that the intent, and why?

**Review by andre-merzky — APPROVED** [review]
We decided tyo merge w/o explicit placement as Theta gives trouble.  We'll open an issue for that.

**Comment by mtitov** [comment]
closed in favor #2502

**Pull request closed without merging**